static/community.svg

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
For both the SVGs we could strip the Inkscape data(In Inkscape: Save As->Plain SVG).

Reply:
Fixed!